index.md

```diff
@@ -19,22 +19,22 @@ ms.assetid:
 <article id="main">
     <section id="hero-content" class="graph">
         <h1>.NET Documentation</h1>
-        <h2>.NET is a development platform for any kind of app or workload, providing key capabilities for building high quality apps quickly. It consists of the common language runtime (CLR) and .NET libraries, which includes APIs that support an extensive range of technologies. .NET provides a managed execution environment, simplified development and deployment, and integration with a variety of programming languages, including C#, F# and Visual Basic.</h2>
+        <h2>.NET is a general purpose development platform that you can use for your next [cloud | desktop | mobile | web | IoT] app or game. It includes the C#, F# and Visual Basic programming languages, an extensive class library and API bindings for popular OSes. Start learning .NET today.</h2>
```

Comment:
nits:
OSes -> operating systems? I think it reads better, especially on a homepage
Should you end the paragraph with an exclamation point?